### PR TITLE
risk merge strategy

### PR DESCRIPTION
**Pull request by dkaman**
this will allow for only risks that have AutoTriage in the comment field to have their history updated even when their status is the same. This will happen if the model suggests to keep something in the triage state. This is not going to happen forever as eventually the model will be making more binary decisions but this may happen for a bit until we get there.

### CODE REVIEW

File: pkg/sdk/model/risk.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ func (r *Risk) Is(status string) bool {
 }
 
 func (r *Risk) Merge(update Risk) {
-	if update.Status != "" && r.Status != update.Status {
+	if (update.Status != "" && r.Status != update.Status) || strings.Contains(update.Comment, "AutoTriage") {
```
Comment (privateducky) [inline pkg/sdk/model/risk.go:40]: why not just check if the comment is empty or not?

File: pkg/sdk/model/risk.go [hunk 2]
```diff
@@ -37,7 +37,7 @@ func (r *Risk) Is(status string) bool {
 }
 
 func (r *Risk) Merge(update Risk) {
-	if (update.Status != "" && r.Status != update.Status) || strings.Contains(update.Comment, "AutoTriage") {
+	if update.Comment != "" {
```
Comment (privateducky) [inline pkg/sdk/model/risk.go:40]: ah - we prob crossed paths on this. I think we need either a comment is there OR the status has changed.

Comment (dkaman) [reply pkg/sdk/model/risk.go:40]: lol this is what i specifically asked to clarify during the meeting this morning, will change again.

Comment (privateducky) [reply pkg/sdk/model/risk.go]: hahahaha well clearly we were on two different planes

Comment (dkaman) [reply pkg/sdk/model/risk.go]: welcome aboard!

**Pull request merged**